spec/ddoc.dd

```diff
@@ -199,11 +199,17 @@ $(H3 $(LNAME2 sections, Sections))
 
 $(P
 The document comment is a series of $(I Section)s.
-A $(I Section) is a name that is the first non-blank character on
+A $(I Section name) is a name that is the first non-blank character on
 a line immediately followed by a ':'. This name forms the section name.
 The section name is not case sensitive.
 )
 
+$(P
+Section names do not contain whitespace. Underscores will be replaced with spaces.
+For example, a section entitled 'New API' would be written with having its first line
+'New_API:'
```

Comment:
"written with having its first line" is rather awkward. And it isn't accurate. The section name doesn't need to be the first line, but at the front of the first line. So something like:

"For example, the first line of a section titled 'New API' would begin with 'New_API:'."

Reply:
Strike "with" and change "would" to "could" makes it both grammatical and semantically accurate.